bms_alerter.py

```diff
@@ -0,0 +1,24 @@
+import bms_alert_definitions as constant
+
+BMS_language = "English"
+SUPPORTED_LANGUAGES = ['English', 'German']
+
+
+def set_language_for_report_and_alerts(selected_language):
+    global BMS_language
+    if selected_language not in SUPPORTED_LANGUAGES:
+        print('\n ALERT!!! Report is not available in', selected_language, '!!!', 'Switching to Default Language')
+    else:
+        BMS_language = selected_language
+
+
+def report_normal_health_status():
+
+    print(constant.battery_health_status_message[BMS_language]['normal'])
```

Comment:
Nice way to keep the display till the end and make that only language-specific :)